components/external_image/external_image.jsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {getImageSrc} from 'utils/post_utils.jsx';
+
+export default class ExternalImage extends React.PureComponent {
+    static propTypes = {
+        children: PropTypes.func.isRequired,
+        enableSVGs: PropTypes.bool.isRequired,
+        hasImageProxy: PropTypes.bool.isRequired,
+        imageMetadata: PropTypes.object,
+        src: PropTypes.string.isRequired,
+    };
+
+    shouldRenderImage = () => {
+        if (this.props.imageMetadata && this.props.imageMetadata.format === 'svg' && !this.props.enableSVGs) {
```

Comment:
Can we just return the not of this expression?

Reply:
Yep, I can change that. There used to be another if statement here as well, so it used to make more sense to have it split up like this